projects/baby_driver/scripts/gpio_get.py

```diff
@@ -0,0 +1,44 @@
+"""Python implementation of the gpio_get function for Babydriver."""
+
+import can_util
+from gpio_port import GpioPort
+from message_defs import BabydriverMessageId
+
+def gpio_get(port, pin):
+    '''
+    Returns the state of the GPIO pin at the given port and pin number as a bool
+
+    The port can be entered as either a string or int value (e.g. 'A' or 0).
+    The pin is an int value
+    '''
+
+    if port < 0 or port >= GpioPort.NUM_GPIO_PORTS:
+        raise ValueError("ERROR: invalid GPIO port")
+    if pin < 0 or pin >= 16:
+        raise ValueError("ERROR: invalid GPIO pin number")
+
+    data = [port, pin]
+    can_util.send_message(
+        babydriver_id=BabydriverMessageId.GPIO_GET_COMMAND,
+        data=data,
+        channel=can_util.default_channel
+    )
+
+    gpio_data_msg = can_util.next_message(
+        babydriver_id=BabydriverMessageId.GPIO_GET_DATA,
+        channel=can_util.default_channel
+    )
+
+    status_msg = can_util.next_message(
+        babydriver_id=BabydriverMessageId.GPIO_GET_DATA,
+        channel=can_util.default_channel
+    )
+
+    if status_msg.msg.data[0] != BabydriverMessageId.STATUS:
+        raise Exception("ERROR: never recieved status message")
+
+    msg_data = gpio_data_msg.msg.data[0]
```

Comment:
A few things about this line:
1. Like above, you don't need to access the `msg` property, just `gpio_data_msg.data` will work
2. It should be `data[1]` since the pin state is in the 2nd uint8; the first uint8 is the babydriver ID.
3. `msg_data` is a bit of a vague name, perhaps `raw_state` would be better.